crates/llm_gateway/src/stream_context.rs

```diff
@@ -53,6 +57,9 @@ impl StreamContext {
             request_id: None,
             start_time: None,
             ttft_duration: None,
+            traceparent: None,
+            ttft_time: None,
```

Comment:
probably not now but are we tracking other latency metrics? e2e for example, we should structure the parameters as a class, not just a parameter

Reply:
this tracks time to first token. This whole code block was moved from prompt filter